R/bundle.R

```diff
@@ -933,50 +937,58 @@ snapshotRDependencies <- function(appDir, implicit_dependencies = c(), verbose =
     name = names(named.repos),
     url = as.character(named.repos),
     contrib.url = contrib.url(named.repos, type = "source"),
-    stringsAsFactors = FALSE)
-
-  # get packages records defined in the lockfile
-  records <- utils::tail(df, -1)
+    stringsAsFactors = FALSE
+  )
 
-  # if the package is in a named CRAN-like repository capture it
+  # Sources are created by packrat:
+  # https://github.com/rstudio/packrat/blob/main/R/pkg.R#L328
```

Comment:
nit: source references against the main branch grow stale as the code evolves; should we link to specific commits or tags?